planning/sampling_based_planner/autoware_bezier_sampler/src/bezier_sampling.cpp

```diff
@@ -58,6 +58,33 @@ std::vector<Bezier> sample(
   }
   return curves;
 }
+Bezier sample(
+  const autoware::sampler_common::State & initial, const autoware::sampler_common::State & final,
+  const double initial_velocity, const double final_velocity, const double acceleration)
+{
+  double distance_initial_to_final = std::sqrt(
+    (initial.pose.x() - final.pose.x()) * (initial.pose.x() - final.pose.x()) +
+    (initial.pose.y() - final.pose.y()) * (initial.pose.y() - final.pose.y()));
+  // Tangent vectors
+  const Eigen::Vector2d initial_tangent_unit(std::cos(initial.heading), std::sin(initial.heading));
+  const Eigen::Vector2d final_tangent_unit(std::cos(final.heading), std::sin(final.heading));
+  // Unit vectors
+  const Eigen::Vector2d initial_normal_unit = initial_tangent_unit.unitOrthogonal();
+  const Eigen::Vector2d final_normal_unit = final_tangent_unit.unitOrthogonal();
+  double initial_tangent_length = initial_velocity * distance_initial_to_final;
+  double final_tangent_length = final_velocity * distance_initial_to_final;
+  double acceleration_length = acceleration * distance_initial_to_final;
```

Comment:
Everything can be made `const` here.
```suggestion
  const double distance_initial_to_final = std::sqrt(
    (initial.pose.x() - final.pose.x()) * (initial.pose.x() - final.pose.x()) +
    (initial.pose.y() - final.pose.y()) * (initial.pose.y() - final.pose.y()));
  // Tangent vectors
  const Eigen::Vector2d initial_tangent_unit(std::cos(initial.heading), std::sin(initial.heading));
  const Eigen::Vector2d final_tangent_unit(std::cos(final.heading), std::sin(final.heading));
  // Unit vectors
  const Eigen::Vector2d initial_normal_unit = initial_tangent_unit.unitOrthogonal();
  const Eigen::Vector2d final_normal_unit = final_tangent_unit.unitOrthogonal();
  const double initial_tangent_length = initial_velocity * distance_initial_to_final;
  const double final_tangent_length = final_velocity * distance_initial_to_final;
  const double acceleration_length = acceleration * distance_initial_to_final;
```